setup.cfg

```diff
@@ -30,7 +30,7 @@ package_dir =
 include_package_data = True
 python_requires = >=3.8
 install_requires =
-    numpy
+    numpy @ git+https://github.com/HaoZeke/numpy.git@fix22648
```

Comment:
Should be also RC

Reply:
I guess there is no requirements for numpy in install section. I checked that chromo built with numpy 1.26.0rc1 works with numpy>1.20.